Source/Core/Core/BootManager.cpp

```diff
@@ -433,11 +433,23 @@ bool BootCore(std::unique_ptr<BootParameters> boot, const WindowSystemInfo& wsi)
   if (StartUp.bWii && DiscIO::IsNTSC(StartUp.m_region) && Config::Get(Config::SYSCONF_PAL60))
     Config::SetCurrent(Config::SYSCONF_PAL60, false);
 
-  // Ensure any new settings are written to the SYSCONF
   if (StartUp.bWii)
   {
-    Core::BackupWiiSettings();
-    ConfigLoaders::SaveToSYSCONF(Config::LayerType::Meta);
+    const bool want_determinism = Movie::IsMovieActive() || NetPlay::IsNetPlayRunning();
+    Core::InitializeWiiRoot(want_determinism);
+
+    // Ensure any new settings are written to the SYSCONF
+    if (!want_determinism)
+    {
+      Core::BackupWiiSettings();
+      ConfigLoaders::SaveToSYSCONF(Config::LayerType::Meta);
+    }
+    else
+    {
+      ConfigLoaders::SaveToSYSCONF(Config::LayerType::Meta, [](const Config::Location& location) {
+        return Config::GetActiveLayerForConfig(location) >= Config::LayerType::Movie;
```

Comment:
Could we add a comment to LayerType to mention that layer types must be listed in reverse search order? Otherwise it's not obvious that the enum order actually matters.